projects/1.16/assets/scp-lockdown/scplockdown/lang/zh_cn.json

```
@@ -0,0 +1,248 @@
+  "block.scplockdown.straight_pipe_black": "黑色管道",
+  "block.scplockdown.straight_pipe_blue": "蓝色管道",
+  "block.scplockdown.straight_pipe_brown": "棕色管道",
+  "block.scplockdown.straight_pipe_cyan": "青色管道",
+  "block.scplockdown.straight_pipe_gray": "灰色管道",
+  "block.scplockdown.straight_pipe_green": "绿色管道",
+  "block.scplockdown.straight_pipe_light_blue": "淡蓝色管道",
+  "block.scplockdown.straight_pipe_lime": "鲜绿色管道",
+  "block.scplockdown.straight_pipe_magenta": "品红色管道",
+  "block.scplockdown.straight_pipe_orange": "橙色管道",
+  "block.scplockdown.straight_pipe_pink": "粉红色管道",
+  "block.scplockdown.straight_pipe_purple": "紫色管道",
+  "block.scplockdown.straight_pipe_red": "红色管道",
+  "block.scplockdown.straight_pipe_silver": "银色管道",
+  "block.scplockdown.straight_pipe_white": "白色管道",
+  "block.scplockdown.straight_pipe_yellow": "黄色管道",
+  "block.scplockdown.sublevel_slab": "横纹半砖",
+  "block.scplockdown.sublevel_stairs": "横纹楼梯",
+  "block.scplockdown.sublevel_wall_a": "横纹墙 A",
+  "block.scplockdown.sublevel_wall_b": "横纹墙 B",
+  "block.scplockdown.sublevel_wall_c": "横纹墙 C",
+  "block.scplockdown.table": "桌子",
+  "block.scplockdown.tjunc_pipe_black": "黑色T型管道",
+  "block.scplockdown.tjunc_pipe_blue": "蓝色T型管道",
+  "block.scplockdown.tjunc_pipe_brown": "棕色T型管道",
+  "block.scplockdown.tjunc_pipe_cyan": "青色T型管道",
+  "block.scplockdown.tjunc_pipe_gray": "灰色T型管道",
+  "block.scplockdown.tjunc_pipe_green": "绿色T型管道",
+  "block.scplockdown.tjunc_pipe_light_blue": "淡蓝色T型管道",
+  "block.scplockdown.tjunc_pipe_lime": "鲜绿色T型管道",
+  "block.scplockdown.tjunc_pipe_magenta": "品红色T型管道",
+  "block.scplockdown.tjunc_pipe_orange": "橙色T型管道",
+  "block.scplockdown.tjunc_pipe_pink": "粉红色T型管道",
+  "block.scplockdown.tjunc_pipe_purple": "紫色T型管道",
+  "block.scplockdown.tjunc_pipe_red": "红色T型管道",
+  "block.scplockdown.tjunc_pipe_silver": "银色T型管道",
+  "block.scplockdown.tjunc_pipe_white": "白色T型管道",
+  "block.scplockdown.tjunc_pipe_yellow": "黄色T型管道",
+  "block.scplockdown.toilet": "马桶",
+  "block.scplockdown.vent": "通风口",
+  "block.scplockdown.ventilation": "通风管道",
+  "block.scplockdown.wall_a": "A型墙",
+  "block.scplockdown.wall_b": "B型墙",
+  "block.scplockdown.wall_c": "C型墙",
+  "block.scplockdown.wall_d": "D型墙",
+  "block.scplockdown.wall_e": "E型墙",
+  "block.scplockdown.wall_f": "F型墙",
+  "block.scplockdown.wall_g": "G型墙",
+  "block.scplockdown.wall_slab": "A型半砖",
+  "block.scplockdown.wall_stairs": "A型楼梯",
+  "block.scplockdown.white_slab": "白色半砖",
+  "block.scplockdown.white_stairs": "白色楼梯",
+  "block.scplockdown.white_wall": "白墙",
+  "block.scplockdown.wood_crate": "木箱",
+  "container.crate": "周转箱",
+  "container.locker": "储物柜",
+  "item.scplockdown.keycard_level1": "一级钥匙卡",
+  "item.scplockdown.keycard_level2": "二级钥匙卡",
+  "item.scplockdown.keycard_level3": "三级钥匙卡",
+  "item.scplockdown.keycard_level4": "四级钥匙卡",
+  "item.scplockdown.keycard_level5": "五级钥匙卡",
+  "item.scplockdown.screwdriver": "改锥",
+  "itemGroup.lockdown_building_blocks": "SCP：封锁 建筑方块",
```

```suggestion
  "itemGroup.lockdown_building_blocks": "SCP：封锁丨建筑方块",
```
【要求】创造模式物品栏应使用汉字`丨`（gǔn）作为分隔符